src/TextField/TextFieldLabel.js

```diff
@@ -26,7 +26,15 @@ export const styleSheet = createStyleSheet('TextFieldLabel', (theme) => {
 }, { index: 5 });
 
 export default function TextFieldLabel(props, context) {
-  const { animated, className: classNameProp, shrink, ...other } = props;
+  const {
+    animated,
+    asteriskClassName,
```

Comment:
Wouldn't it be better for the `TextFieldLabel` to own this style instead of the parent?